spec/factories/books.rb

```diff
@@ -1,10 +1,35 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: books
+#
+#  id              :bigint           not null, primary key
+#  title           :string
+#  author          :string
+#  cover_image_url :string
+#  page_count      :integer
+#  publisher       :string
+#  synopsis        :text
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#
 FactoryBot.define do
   factory :book do
-    title { "MyString" }
-    author { "MyString" }
-    cover_image_url { "MyString" }
-    page_count { 1 }
-    publisher { "MyString" }
-    synopsis { "MyText" }
+    title { "The Lord of the Rings" }
+    author { "J. R. R. Tolkien" }
+    cover_image_url { "https://images-na.ssl-images-amazon.com/images/I/51r6XIPWmoL._SX331_BO1,204,203,200_.jpg" }
+    page_count { "" }
+    publisher { "Houghton Mifflin Harcour" }
+    synopsis { "In ancient times the Rings of Power were crafted by the Elven-smiths, and Sauron, the Dark Lord, forged the One Ring, filling it with his own power so that he could rule all others. But the One Ring was taken from him, and though he sought it throughout Middle-earth, it remained lost to him. After many ages it fell by chance into the hands of the hobbit Bilbo Baggins." }
+  end
+
+  factory :random_book, class: Book do
```

Comment:
I suggest this as your primary factory for books. To make sure that your tests are robust because it is tested with random data.